Add icon button to open console in bottom right of start menu

Robyt3 opened the pull request:
Add a button with the "terminal" icon in the bottom right of the start menu to open the local console to ensure that the local console is usable also when no physical keyboard (with F-keys) is available.

![screenshot_2024-08-27_19-39-51](https://github.com/user-attachments/assets/300b8cad-ece3-4ca5-a19d-8f1da0337f2b)

## Checklist

- [X] Tested the change ingame
- [X] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

def- reviewed (commented):
It somehow doesn't fit in that location based on the other buttons. How about below Settings instead? I guess someone needs to draw a picture then.

Robyt3 commented:
It doesn't really feel important enough to be in the center to me.

Maybe in the bottom right with an icon?

![screenshot_2024-08-27_19-39-51](https://github.com/user-attachments/assets/300b8cad-ece3-4ca5-a19d-8f1da0337f2b)

ChillerDragon commented:
I agree with rossbit. The console does not belong to the center. The icon on the bottom right is really cool.
@Robyt3 do you also plan adding way to open the console while being in game?

Robyt3 commented:
I updated the PR so the console button is in the bottom right of the start menu now.

> @Robyt3 do you also plan adding way to open the console while being in game?

I'm not sure where I would put those buttons though. For #8632 I added another row to the ingame menu when touch controls are enabled.
